src/lib/components/SelectField/SelectField.jsx

```diff
@@ -51,7 +57,7 @@ export const SelectField = React.forwardRef((props, ref) => {
       <div
         className={classNames(
           styles.label,
-          !isLabelVisible && styles.isLabelHidden,
+          (!isLabelVisible || inputGroupContext) && styles.isLabelHidden,
```

Comment:
Please document these conditions in the component API, e.g. for label visibility:

`Automatically set to 'false' when the component is rendered within InputGroup.` (⬅️ please use backticks instead of single quotes)

And for validation text:

`Validation text is never rendered when the component is placed into an InputGroup. Instead, the InputGroup itself renders all validation texts of its nested components.`

Or something like that…

Please, document it in **all affected components**, not only `SelectField`. Thank you!